packaging/ubuntu-14.04/snapd.postrm

```diff
@@ -4,13 +4,19 @@ set -e
 
 systemctl_stop() {
     unit="$1"
-    for i in $(seq 10); do
+    if ! systemctl is-active -q "$unit"; then
+        echo "$unit is stopped."
+        return
```

Comment:
The unit may become active at this time and we will just not catch that.

Could we always attempt to stop it and then continue to the waiting loop below?